File: @LV1/1.3/sibice/sibice.cpp
#include <cmath>
#include <cstdio>

int main() {
    int n;
    double w, h, d;
    scanf("%d %lf %lf", &n, &w, &h);

    d = sqrt(w * w + h * h);

    for (int i = 0; i < n; i++) {
        double k;
        scanf("%lf", &k);

        if (k <= d) {
            puts("DA");
        } else {
            puts("NE");
        }
    }
}

File: LV1/1.3/sibice/sibice.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n int
	var w, h float64
	fmt.Scanf("%d %f %f", &n, &w, &h)

	d := math.Sqrt(w*w + h*h)

	for i := 0; i < n; i++ {
		var k float64
		fmt.Scanf("%f", &k)

		if k <= d {
			fmt.Println("DA")
		} else {
			fmt.Println("NE")
		}
	}
}
